werkit/compute/graph/schema.ts

```diff
@@ -0,0 +1,17 @@
+export type ValueType = 'Number' | 'Point' | 'Measurement'
```

Comment:
I wonder if we should reuse the following typescript types from goldie-data-layer:

* for Point: [NamedPoint](https://github.com/curvewise/frontend/blob/main/published-packages/common-types/src/geometry.ts#L8-L11) or [Vector3](https://github.com/curvewise/frontend/blob/main/published-packages/common-types/src/geometry.ts#L1)
* for Measurement: [Polyline](https://github.com/curvewise/frontend/blob/main/published-packages/common-types/src/geometry.ts#L3-L6)
* for Number, this can just be the typescript `number` type primitive